preppal-be/routes/reviewApi.ts

```diff
@@ -0,0 +1,129 @@
+const expressReviewApi = require("express");
+const configReviewApi = require("../configs/secrets.ts");
+
+const Review = require("../models/review.ts");
+const Author = require("../models/user.ts");
+const Recipe = require("../models/recipe.ts");
+
+const routerReviewApi = expressReviewApi.Router();
+
+
+/**
+ * GET - Get all recipes
+ */
+routerReviewApi.get("/", async (req, res) => {
+    try {
+        const reviews = await Review.find();
+        res.status(200).json(reviews);
+    }
+    catch (error) {
+        console.error(error.message);
+        res.status(500).send("Server error.");
+    }
+});
+
+/**
+ * GET - Get reviews by recipeId
+ */
+routerReviewApi.get("/:recipeId", async (req, res) => {
+    try {
+        const reviews = await Review.findOne({ recipeId: req.params.recipeId });
+        res.status(200).json(reviews);
```

Comment:
probably only want to review only the reviews?

no harm in returning the recipe id as well I guess